src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -14,77 +14,80 @@ In this recipe, you will learn how to dynamically import your images using Vite'
 
 1. Create a new `assets` folder under the `src` directory and add your images inside that new folder. 
 
-<FileTree>
-- src/
-  - assets/
-    - avatar-1.jpg
-    - avatar-2.png
-    - avatar-3.jpeg
-</FileTree>
-
-:::note
-`assets` is a popular folder name convention for placing images but you are free to name the folder whatever you like.
-::: 
+    <FileTree>
+    - src/
+      - assets/
+        - avatar-1.jpg
+        - avatar-2.png
+        - avatar-3.jpeg
+    </FileTree>
+
+    :::note
+    `assets` is a popular folder name convention for placing images but you are free to name the folder whatever you like.
+    ::: 
 
 2. Create a new Astro component for your card and import the `<Image />` component.
 
     ```astro title="src/components/MyCustomCardComponent.astro" 
     ---
-     import { Image } from 'astro:assets';
+    import { Image } from 'astro:assets';
     ---
     ```
 
 3. Specify the `props` that your component will receive in order to display the necessary information on each card. You can optionally define their types, if you are using TypeScript in your project.
 
-    ```astro title="src/components/MyCustomCardComponent.astro" ins={4-11}
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={4-9, 10-11}
```

Comment:
This looks to be doing the same thing. Everything should be correctly highlighted using the existing `ins` value.
```suggestion
    ```astro title="src/components/MyCustomCardComponent.astro" ins={4-11}
```